Formula/porg.rb

```diff
@@ -0,0 +1,26 @@
+class Porg < Formula
+  desc "Source Code Package Organizer"
+  homepage "http://porg.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/porg/porg-0.10.tar.gz"
+  sha256 "48f8433193f92097824ed7a72c4babafb29dc2ffe60e7df3635664f59f09cedd"
+
+  option "without-grop", "Don't install Grop, which is the graphic interface of porg."
```

Comment:
Make this:
```ruby
option "without-gtkmm3", "Don't install Grop, which is the graphic interface of porg."
```
and then:
```ruby
depends_on "gtkmm3" => :recommended
```
Our dependency system doesn't entirely love `if build.with?` statements outside of `def install`. It works, but we prefer to avoid them where possible.